tests/test_core_to_contrib.py

```diff
@@ -810,12 +812,21 @@
         "airflow.providers.amazon.aws.sensors.sqs.SQSSensor",
         "airflow.contrib.sensors.aws_sqs_sensor.SQSSensor",
     ),
+]
 
+PROTOCOLS = [
+    (
+        "airflow.providers.amazon.aws.hooks.batch_client.AwsBatchProtocol",
+        "airflow.contrib.operators.awsbatch_operator.BatchProtocol",
+    ),
 ]
-ALL = HOOK + OPERATOR + SENSOR
+
+
+ALL = HOOK + OPERATOR + SENSOR + PROTOCOLS
+
 RENAMED_HOOKS = [
     (old_class, new_class)
-    for old_class, new_class in ALL
+    for old_class, new_class in HOOK + OPERATOR + SENSOR
```

Comment:
Why did you change this?

Reply:
`ALL` now includes the new `PROTOCOLS`, but the latter do not behave like the rest of the classes in `RENAMED_HOOKS`.  To fully get it, checkout the branch, use `ALL` and run this test to see that the `PROTOCOLS` do not pass the same set of tests because the `PROTOCOLS` are not instantiated and they do not call `super` from an init.  I tried various ways to "make" them conform, but decided they are a category all of their own and they need their own test for the deprecated warning.  Open to suggestions on how this could be improved, but obviously it wasn't the main focus of this PR to start with, so this is all ad-hoc improvements.

Reply:
Since @nuclearpinguin asked for the migration of the protocol to get tested and he approved these changes, I'll assume this is good enough to resolve this.

Reply:
Ah I get it. Thanks.